addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -431,6 +432,97 @@ QUnit.test('show message subject if thread is mailing channel', async function (
     );
 });
 
+QUnit.test('basic rendering of canceled notification', async function (assert) {
+    assert.expect(9);
+
+    this.data['mail.channel'].records.push({ id: 11 });
+    this.data['res.partner'].records.push({ id: 12, name: "Someone" });
+    this.data['mail.message'].records.push({
+        id: 10,
+        channel_ids: [11],
+        message_type: 'email',
+        model: 'mail.channel',
+        notification_ids: [11],
+        res_id: 11,
+    });
+    this.data['mail.notification'].records.push({
+        id: 11,
+        failure_type: 'SMTP',
+        mail_message_id: 10,
+        notification_status: 'canceled',
+        notification_type: 'email',
+        res_partner_id: 12,
+    });
+    await this.start();
+    const threadViewer = await this.env.models['mail.thread_viewer'].create({
+        hasThreadView: true,
+        thread: [['insert', {
+            id: 11,
+            model: 'mail.channel',
+        }]],
+    });
+    await this.afterEvent({
+        eventName: 'o-component-message-list-thread-cache-changed',
+        func: () => {
+            this.createThreadViewComponent(threadViewer.threadView);
+        },
+        message: "should wait until thread view messages are loaded",
```

Comment:
Comment should tell what resolved `await` means, not what the `await` does.
Something like _"Thread become loaded with messages"_.

Also, not expecting changes in context of this PR, but the event name + payload is not expressive enough with what the test is actually waiting for. Reason for that is because of reliance on an heuristics, which work right now but usually heuristics are bad.